### PR TITLE
Added polish translation

### PR DESCRIPTION
**Pull request by zneix**
I am still unsure about few strings, I will have to ask friends for revisions.
If anyone wanna review this, [here](https://github.com/flex3r/DankChat/blob/master/app/src/main/res/values/strings.xml) are original strings.

### CODE REVIEW

File: app/src/main/res/values-pl-rPL/strings.xml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    <string name="login">Zaloguj się</string>
+    <string name="reload_emotes">Przeładuj emotki</string>
+    <string name="reconnect">Połącz ponownie</string>
+    <string name="clear_chat">Wyczyść czat</string>
+    <string name="add_dialog_title">Dodaj kanał</string>
+    <string name="dialog_ok">OK</string>
+    <string name="dialog_cancel">Anuluj</string>
+    <string name="dialog_dismiss">Zignoruj</string>
+    <string name="error_dialog_copy">Kopiuj</string>
+    <string name="error_dialog_title">Złapany błąd: %1$s</string>
```
Comment (Heryin) [inline app/src/main/res/values-pl-rPL/strings.xml]: Inconsistency with line 120, change to "Złapany wyjątek: %1$s"

File: app/src/main/res/values-pl-rPL/strings.xml [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    <string name="system_message_disconnected">Rozłączono</string>
+    <string name="system_message_no_history">Nie widzisz historii wiadomości ponieważ zrezygnowałeś z tej usługi.</string>
+    <string name="timed_out_message">&lt; Wiadomość usunięta ></string>
+    <string name="multi_entry_header_regex">Regex</string>
+    <string name="multi_entry_header_match_user">Sprawdzaj nazwy użytkowników</string>
+    <string name="multi_entry_add_entry">Dodaj wpis</string>
+    <string name="multi_entry_remove_content_description">Usuń wpis</string>
+    <string name="multi_entry_hint">Wpis</string>
+    <string name="login_expired">Logowanie wygasło! Spróbuj zalogować się ponownie.</string>
+    <string name="message_history_disclaimer_title">Message history disclaimer</string>
```
Comment (Heryin) [inline app/src/main/res/values-pl-rPL/strings.xml:69]: Not translated to polish
Suggested translation: "Informacja o historii wiadomości"

File: app/src/main/res/values-pl-rPL/strings.xml [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    <string name="preference_debug_mode_title">Tryb debugowania</string>
+    <string name="preference_debug_mode_summary">Dostarcza informacje o wszelkich złapanych wyjątkach</string>
+    <string name="preference_timestamp_format_title">Format znaczka czasu</string>
+    <string name="preference_tts_title">Włącz TTS</string>
+    <string name="preference_tts_summary">Czyta na głos wiadomości aktywnego kanału</string>
+    <string name="preference_tts_queue_title">Tryb kolejki</string>
+    <string name="preference_tts_queue_add">Czyta każdą wiadomość używając kolejki</string>
+    <string name="preference_tts_queue_flush">Czyta tylko najnowszą wiadomość</string>
+    <string name="preference_tts_message_format_message">Czyta tylko wiadomość</string>
+    <string name="preference_tts_message_format_combined">Czyta nazwę autora i wiadomość</string>
```
Comment (Heryin) [inline app/src/main/res/values-pl-rPL/strings.xml:128]: Change "czyta" to "odczytuje"

Comment (zneix) [reply app/src/main/res/values-pl-rPL/strings.xml]: Fixed in `eb7f80b`

File: app/src/main/res/values-pl-rPL/strings.xml (hunk 2 above)
Comment (zneix) [reply app/src/main/res/values-pl-rPL/strings.xml]: Fixed in `eb7f80b`

File: app/src/main/res/values-pl-rPL/strings.xml (hunk 1 above)
Comment (zneix) [reply app/src/main/res/values-pl-rPL/strings.xml]: While "wyjątek" is a bit more professional, I think "błąd" is something easier to understand for most end-users, so I will change the other expression in line 120. to "błąd". Resolved in `eb7f80b`

**Comment by flex3r** [comment]
Thanks for the contribution! 👍

**Pull request merged**